orangecontrib/prototypes/widgets/owfactoranalysis.py

```diff
@@ -24,39 +37,97 @@ class Inputs:
     class Outputs:
         sample = Output("Sampled Data", Table)
 
-    settingsHandler = settings.DomainContextHandler()   # TODO puciga
     n_components = settings.ContextSetting(1)
-    commitOnChange = settings.Setting(0)        #TODO puciga
+    setting_for_rotation = settings.Setting(Rotation.NoRotation)
```

Comment:
`setting_for_rotation` is not a good name. Skip `setting_for`; `rotation` will probably suffice. (This is almost like naming you variable e.g. `variable_with_index` instead of just `index`.)